src/app/clients/ActionBar.tsx

```diff
@@ -109,18 +118,40 @@ const ActionBar: React.FC<Props> = ({ selected, data }) => {
                 onSubmit={submitStatus}
                 errorText={modalError}
             />
-            <ActionBarModal
-                isOpen={actionModal}
-                onClose={() => {
-                    setActionModal(false);
-                    setModalError(null);
-                }}
-                data={selectedData}
-                header={selectedAction ?? "Apply Action"}
-                headerId="action-modal-header"
-                onSubmit={submitAction}
-                errorText={modalError}
-            />
+            {shippingLabelsModal ? (
```

Comment:
A good QOL change would be to increase the checkbox size/ clickable radius, as it is really hard to click them atm

Reply:
Will deal with this in a future ticket